qase-pytest/src/qaseio/pytest/plugin.py

```diff
@@ -81,7 +82,7 @@ def drop_run_id():
     def pytest_collection_modifyitems(self, session, config, items):
         # Filter test cases based on ids
         if config.option.qase_testops_plan_id or config.option.qase_testops_run_id:
-            items[:] = [item for item in items if item.get_closest_marker('qase_id') and
+            items[:] = [item for item in items if item.get_closest_marker('qase_id') and self.execution_plan and
```

Comment:
Is it no better to add it one line above as it will not change per item:
```
if self.execution_plan and (config.option.qase_testops_plan_id or config.option.qase_testops_run_id):
            items[:] = [item for item in items if item.get_closest_marker('qase_id') and
             ....
```